src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -262,7 +264,15 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 				let outputElement = <HTMLElement>this.output.nativeElement;
 				outputElement.innerHTML = this.markdownResult.element.innerHTML;
 				this.addUndoElement(outputElement.innerHTML);
-
+				// Find cell editor provider via cell guid to get markdown editor height
+				let cellEditorProvider = this.markdowncodeCell.find(c => c.cellGuid() === this.cellModel.cellGuid);
+				if (cellEditorProvider && this.markdownMode) {
+					let editor = cellEditorProvider.getEditor() as QueryTextEditor;
```

Comment:
Since getEditor() already returns a QueryTextEditor, do we still need the 'as QueryTextEditor' here?

Reply:
Maybe naming this markdownEditor can make it easier to differentiate from _editorHeight.